doajtest/helpers.py

```diff
@@ -50,7 +94,7 @@ def _make_and_push_test_context(self, path="/", acc=None):
         ctx = self.app_test.test_request_context(path)
         ctx.push()
         if acc is not None:
-            acc.save(blocking=True)
+            # acc.save(blocking=True)
```

Comment:
can we get away with not saving our user? 

we have this in `core.py` which touches the index on login (`LoginManager.reload_user()`
```python
@login_manager.user_loader
def load_account_for_login_manager(userid):
    """
    ~~LoginManager:Feature->Account:Model~~
    :param userid:
    :return:
    """
    from portality import models
    out = models.Account.pull(userid)
    return out
    ```

Reply:
Not sure yet, I just took it out for the moment so that there was no elasticsearch stuff in the main test superclass.  I think it'll have to go back in somehow, but I'm not sure how yet